backend/.env.dist

```diff
@@ -52,6 +52,9 @@ EMAIL_CODE_REQUEST_TIME=10
 # Webhook
 WEBHOOK_ELOPAGE_SECRET=secret
 
+# EventProtocol
+EVENT_PROTOCOL_ENABLED=true
```

Comment:
We need this in the `.env.template` as well. See my documentation on `.env`

Reply:
ok

Reply:
done